A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/BAT/A320_Neo_BAT.css

```diff
@@ -49,21 +51,35 @@
   font-weight: 900;
   font-style: normal; }
 
+
+
+  /* JZ added a Special font for the Voltage display */
+  @font-face {
+    font-family: "Digitalv";
+    src: url("/Fonts/wwDigital.ttf") format("truetype"); /* New font type for the v in Volts. Curiously, the font is not loaded in the sim??? */
```

Comment:
Remove comment

Reply:
Well since this is a collaborative project, bette to inform the others of what has changed, no?

Reply:
Agreed, comment is fine